olp-cpp-sdk-core/src/client/OlpClient.cpp

```diff
@@ -197,10 +197,20 @@ std::shared_ptr<http::NetworkRequest> OlpClient::CreateRequest(
     const std::multimap<std::string, std::string>& header_params,
     const std::shared_ptr<std::vector<unsigned char>>& post_body,
     const std::string& content_type) const {
+  auto http_verb = GetHttpVerb(method);
+  return CreateRequest(path, http_verb, query_params, header_params, post_body,
+                       content_type);
+}
+
+std::shared_ptr<http::NetworkRequest> OlpClient::CreateRequest(
+    const std::string& path, olp::http::NetworkRequest::HttpVerb http_verb,
```

Comment:
```suggestion
    const std::string& path, http::NetworkRequest::HttpVerb http_verb,
```
Same for all others below.